lib/src/package_config_json.dart

```diff
@@ -108,12 +108,12 @@ PackageConfig parsePackageConfigJson(
     return null;
   }
 
-  Package /*?*/ parsePackage(Map<String, dynamic> entry) {
-    String /*?*/ name;
-    String /*?*/ rootUri;
-    String /*?*/ packageUri;
-    String /*?*/ languageVersion;
-    Map<String, dynamic> /*?*/ extraData;
+  Package? parsePackage(Map<String, dynamic> entry) {
```

Comment:
We have been going with `Map<String, Object?>` for all the other packages because it makes the potential error conditions more explicit via explicit casts instead of dynamic calls.

Reply:
I'll remove every instance of `dynamic`. And good riddance :)